test/input_files/poisson_periodic_2d_x.in

```diff
@@ -63,8 +63,8 @@
 
    [./Generation]
       dimension = '2'
-      n_elems_x = '20'
-      n_elems_y = '20'
+      n_elems_x = '10'
```

Comment:
Could you please add a comment here that we're overriding on the command line to 20x20 to test the command-line-override capability? I could see future me wasting far too much time missing that connection if I came back to monkey with this test.